src/chapters/sql-part2/sql-part2.rst

```diff
@@ -4,6 +4,420 @@ SQL Part 2 - String and Date Functions
+     - 8
+   * -
+     - 
+     - **SELECT YEAR('2022-08-21');**
+     - 2022
+   * - `CONVERT <https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15#j-using-convert-with-datetime-data-in-different-formats>`__.
+     - Paired with GETDATE to `convert other data types into dates and times <https://learn.microsoft.com/en-us/sql/t-sql/data-types/date-transact-sql?view=sql-server-ver15#converting-date-to-other-date-and-time-types>`__ based on the desired format.
+     - **SELECT CONVERT(varchar, GETDATE());**
+     - Jan  5 2022  7:25PM
+   * - 
+     - These examples convert the server date into various formats. The first example is the default format, the second example is format number 7. 
+     - **SELECT CONVERT(varchar, GETDATE(), 7);**
+     - Jan 05, 22
+   * - `FORMAT <https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql?view=sql-server-ver15>`__.
+     - Used to set the format or reformat dates. This example is using the following date information: Jan  5, 2022  7:38PM. The first example the server date is formatted date, month, year. The time is not requested, so it is not returned.
+     - **SELECT FORMAT(GETDATE(), 'dd/MM/yyyy');**
+     - 05/01/2022
+   * - 
+     - The second example requests only the time in hours and minutes. Note that 'MM' is used for months and 'mm' is used for minutes.
+     - **SELECT FORMAT(GETDATE(), 'hh:mm');**
+     - 07:38
+
+Part 3: `More Aggregations and Miscellany <https://docs.microsoft.com/en-us/sql/t-sql/functions/aggregate-functions-transact-sql?view=sql-server-ver15>`__.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+| The following examples are using the BooksDB.  
+
+.. list-table::
+   :align: left
+   :header-rows: 1
+   :widths: 15 30 35 20
+   
+   * - **Function**
+     - **Definition**
+     - **Syntax**
+     - **Value Returned**
+   * - `SUM <https://docs.microsoft.com/en-us/sql/t-sql/functions/sum-transact-sql?view=sql-server-ver15>`__.
+     - Returns sum of all values or DISTINCT values
+     - **SELECT SUM(ratings_2) FROM BooksDB.dbo.books;**
+     - 31108850
+   * - `MAX <https://docs.microsoft.com/en-us/sql/t-sql/functions/max-transact-sql?view=sql-server-ver15>`__.
+     - Returns the highest value
+     - **SELECT MAX(ratings_2) FROM BooksDB.dbo.books;**
+     - 436802
+   * - `MIN <https://docs.microsoft.com/en-us/sql/t-sql/functions/min-transact-sql?view=sql-server-ver15>`__.
+     - Returns the lowest value
+     - **SELECT MIN(ratings_2) FROM BooksDB.dbo.books;**
+     - 30
+   * - `IS NULL <https://docs.microsoft.com/en-us/sql/t-sql/queries/is-null-transact-sql?view=sql-server-ver15>`__.
+     - Determines whether value is null, often used as a condition with WHERE
+     - Note in this example we asked for the TOP 5 rows, but only 3 qualified and were returned
+     - 
+  
+.. list-table::
```

Comment:
This can be a code block

Reply:
I've updated the formatting for the entire Reading Section.